### DISCUSSION TITLE
placing the files under 'src' Directory

### DISCUSSION THREAD

**Discussion by jbryan11** [vercel/next.js#13930] [Help] [answered]
Hi! 
 Sorry to bother you bout this small matter but I wanna clarify some things regards to  the src directory. 
 I had a custom _app and _document js file and I want to know if these two would be under the src directory or under src/pages directory? If I put these just under src directory not src/page directory, will these two still override the default _app and _document js file?

**Comment by timneutkens** [comment, marked as answer]
You have to put `_document.js` / `_app.js` under either the root `pages` directory or `src/pages`
